fuzz/brontide/random_actone.go

```diff
@@ -11,7 +11,7 @@ import (
 func Fuzz_random_actone(data []byte) int {
```

Comment:
Don't forget to sign the commit.

Reply:
Donezo, this comment actually motivated me to un-expire my key